### ISSUE TITLE
Uninhabited Houses on 1911 Census

### ISSUE THREAD

**Issue by cmahoneynz** [FreeUKGen/FreeCENMigration#1756]
Can we have a look at allowing uninhabited houses/buildings to be entered on the 1911 census.

Was told that they were excluded as there were not supposed to be uninhabited houses on 1911.  However on a 1911 piece that I had someone start transcribing today has six pages complete with schedule numbers, but nothing entered on them other than notes from the enumerator which include why the schedule had not been used.  And this is not isoloated to this piece or county.  Just about every 1911 piece I have looked at across four different counties includes at least one numbered schedule page which has not been filled out.

In addition, if one where to look at the enumeator list of properties for the piece, there are those either unoccupied on the night, or normally not occupied at night which, given any earlier year would have been included as an uninhabited house/building.  And I can come up with good reasons on why they should be included.

**Comment by PatReynoldsFUG** [comment]
Needs adressing as 1911 being worked on now.

**Comment by PatReynoldsFUG** [comment]
Yes, should be included as very helpful (when they are present) for those working on house history, or trying to work out if an unnumbered house with is the same as one in a previous/next census and thus the John Smith as sole occupant is the same John Smith as the occupant in another census with slightlly different details.

**Comment by PatReynoldsFUG** [comment]
Needs review to agree what is to be done to implement this.

**Comment by cmahoneynz** [comment]
Inclusion can also explain what would otherwise be missing schedule numbers.

Another important reason on why they should be there is that it can give a better indication of where business's are.  For example, one Nottinghasm 1911 piece I looked at had quite a high number of people working for a particular manufacturing company (which I could not find when checked on Google).  Being a factory not normally occupied at night, they were probably not given a census form to return, but included in the enumeator's summary of all the address's contained in the piece is the location of the factory
